nixos/modules/services/web-apps/gancio.nix

```
@@ -222,8 +225,19 @@ in
           ${concatMapStringsSep "\n" (p: "ln -sf ${p} plugins/") cfg.plugins}
         '';
 
+        # Set permission so that nginx can use gancio socket
+        # (gancio/nuxt does not support systemd socket activation)
+        postStart = ''
+          until [ -S ${cfg.settings.server.socket} ]; do
+            echo Waiting for ${cfg.settings.server.socket}
+            sleep 1
+          done
+          chmod 0660 ${cfg.settings.server.socket}
```

Just in case someone puts spaces in their socket path names

```suggestion
          until [ -S '${cfg.settings.server.socket}' ]; do
            echo Waiting for '${cfg.settings.server.socket}'
            sleep 1
          done
          chmod 0660 '${cfg.settings.server.socket}'
```